autograph.yaml

```diff
@@ -31,6 +31,12 @@ statsd:
 #    maxidleconns: 10
 #    monitorpollinterval: 10s
 
+# When the DB or HSM is enabled use these timeouts for the
+# /__heartbeat__ handler
+heartbeat:
+    hsmchecktimeout: 10ms
+    dbchecktimeout: 15ms
```

Comment:
isn't that a bit aggressive and prone to failing when the db or hsm slow down during maintenance?
I feel like 10x, even 100x, these values would still provided the functionality without the flakiness.